diff --git a/apps/github-app/package.json b/apps/github-app/package.json
index 70220582d..7a81511b5 100644
--- a/apps/github-app/package.json
+++ b/apps/github-app/package.json
@@ -17,7 +17,7 @@
     "test": "jest"
   },
   "dependencies": {
-    "axios": "^0.21.1",
+    "axios": "^1.7.2",
     "body-parser": "^1.20.1",
     "dotenv": "^16.0.3",
     "express": "^4.18.2",
diff --git a/client/packages/design/components/VirtualChatList/index.tsx b/client/packages/design/components/VirtualChatList/index.tsx
index b15ce4577..25fb0064c 100644
--- a/client/packages/design/components/VirtualChatList/index.tsx
+++ b/client/packages/design/components/VirtualChatList/index.tsx
@@ -78,5 +78,5 @@ type VirtualChatListInterface = typeof InternalVirtualChatList & React.FC;
 
 export const VirtualChatList: VirtualChatListInterface = React.memo(
   InternalVirtualChatList
-) as any;
+) as VirtualChatListInterface;
 VirtualChatList.displayName = 'VirtualChatList';
diff --git a/client/shared/api/socket.ts b/client/shared/api/socket.ts
index 36aed3899..131930544 100644
--- a/client/shared/api/socket.ts
+++ b/client/shared/api/socket.ts
@@ -136,39 +136,33 @@ export class AppSocket {
 
     // 网络状态管理
     socket.on('connect', () => {
-      console.log('连接成功');
       closeConnecting();
 
       sharedEvent.emit('updateNetworkStatus', 'connected');
     });
     socket.on('connecting', (data) => {
-      console.log('正在连接');
 
       showConnecting();
 
       sharedEvent.emit('updateNetworkStatus', 'reconnecting');
     });
     socket.on('disconnect', (data) => {
-      console.log('与服务器的链接已断开');
       showErrorToasts(t('与服务器的链接已断开'));
       closeConnecting();
       sharedEvent.emit('updateNetworkStatus', 'disconnected');
     });
     socket.on('connect_error', (data) => {
-      console.log('连接失败');
       showErrorToasts(t('连接失败'));
       closeConnecting();
       sharedEvent.emit('updateNetworkStatus', 'disconnected');
     });
 
     socket.io.on('reconnect', (data) => {
-      console.log('重连成功');
 
       closeConnecting();
       sharedEvent.emit('updateNetworkStatus', 'connected');
     });
     socket.io.on('reconnect_attempt', (data) => {
-      console.log('重连中...');
       showConnecting();
       sharedEvent.emit('updateNetworkStatus', 'reconnecting');
     });
diff --git a/client/shared/i18n/index.ts b/client/shared/i18n/index.ts
index da3318f20..b4bb52a59 100644
--- a/client/shared/i18n/index.ts
+++ b/client/shared/i18n/index.ts
@@ -29,7 +29,6 @@ i18next
       loadPath: '/locales/{{lng}}/{{ns}}.json',
       allowMultiLoading: false,
       addPath: (...args: any[]) => {
-        console.log('Lost translate:', ...args);
       },
     },
     react: {
diff --git a/client/shared/package.json b/client/shared/package.json
index 6e49b7d14..5a45a68b6 100644
--- a/client/shared/package.json
+++ b/client/shared/package.json
@@ -11,7 +11,7 @@
     "@tanstack/query-async-storage-persister": "4.32.6",
     "@tanstack/react-query": "4.33.0",
     "@tanstack/react-query-persist-client": "4.33.0",
-    "axios": "^0.21.1",
+    "axios": "^1.7.2",
     "crc": "^3.8.0",
     "dayjs": "^1.10.6",
     "eventemitter-strict": "^1.0.1",
diff --git a/client/shared/redux/setup.ts b/client/shared/redux/setup.ts
index 15b2c2ec3..ea21b0465 100644
--- a/client/shared/redux/setup.ts
+++ b/client/shared/redux/setup.ts
@@ -50,7 +50,6 @@ export function setupRedux(socket: AppSocket, store: AppStore) {
  * 初始化数据
  */
 function initial(socket: AppSocket, store: AppStore) {
-  console.log('初始化Redux上下文...');
 
   // 立即请求加入房间
   socket
diff --git a/client/web/package.json b/client/web/package.json
index 04da01158..afd34460c 100644
--- a/client/web/package.json
+++ b/client/web/package.json
@@ -34,7 +34,7 @@
     "ahooks": "^3.7.4",
     "antd": "^4.24.13",
     "array-move": "3.0.1",
-    "axios": "^0.21.4",
+    "axios": "^1.7.2",
     "bytemd": "^1.21.0",
     "clsx": "^1.2.1",
     "compressorjs": "^1.1.1",
diff --git a/client/web/src/components/ChatBox/ChatInputBox/usePasteHandler.tsx b/client/web/src/components/ChatBox/ChatInputBox/usePasteHandler.tsx
index d32b2016b..219e6a9eb 100644
--- a/client/web/src/components/ChatBox/ChatInputBox/usePasteHandler.tsx
+++ b/client/web/src/components/ChatBox/ChatInputBox/usePasteHandler.tsx
@@ -34,7 +34,6 @@ export function usePasteHandler() {
       }; // for get data later, because event is sync
 
       if (handlers.length === 1) {
-        console.log(`Running paste handler: ${handlers[0].name}`);
         event.stopPropagation();
         event.preventDefault();
         handlers[0].handler(data, context);
@@ -52,7 +51,6 @@ export function usePasteHandler() {
                 key={h.name}
                 className="bg-black bg-opacity-40 hover:bg-opacity-80 rounded px-2 py-1 cursor-pointer"
                 onClick={() => {
-                  console.log(`Running paste handler: ${h.name}`);
                   h.handler(data, context);
                   setInner(null);
                 }}
diff --git a/client/web/src/components/ImageUploader.tsx b/client/web/src/components/ImageUploader.tsx
index f3ac2d066..b73543f89 100644
--- a/client/web/src/components/ImageUploader.tsx
+++ b/client/web/src/components/ImageUploader.tsx
@@ -29,12 +29,10 @@ export const ImageUploader: React.FC<ImageUploaderProps> = React.memo(
           usage: props.usage,
           onProgress(percent) {
             const uploadProgress = Number((percent * 100).toFixed());
-            console.log(`进度:${uploadProgress}`);
             setUploadProgress(uploadProgress);
           },
         });
 
-        console.log('上传成功', fileInfo);
         props.onUploadSuccess(fileInfo);
       },
       []
diff --git a/client/web/src/components/popover/UserPopover/GroupUserPopover.tsx b/client/web/src/components/popover/UserPopover/GroupUserPopover.tsx
index 6bd00065b..2d9f173bc 100644
--- a/client/web/src/components/popover/UserPopover/GroupUserPopover.tsx
+++ b/client/web/src/components/popover/UserPopover/GroupUserPopover.tsx
@@ -76,7 +76,6 @@ export const GroupUserPopover: React.FC<{
   useEffect(() => {
     if (userInfo.avatar) {
       fetchImagePrimaryColor(userInfo.avatar).then((rgba) => {
-        console.log('fetchImagePrimaryColor', rgba);
       });
     }
   }, [userInfo.avatar]);
diff --git a/client/web/src/components/popover/UserPopover/PersonalUserPopover.tsx b/client/web/src/components/popover/UserPopover/PersonalUserPopover.tsx
index 27cba6869..95c432c4e 100644
--- a/client/web/src/components/popover/UserPopover/PersonalUserPopover.tsx
+++ b/client/web/src/components/popover/UserPopover/PersonalUserPopover.tsx
@@ -62,7 +62,6 @@ export const PersonalUserPopover: React.FC<{
   useEffect(() => {
     if (userInfo.avatar) {
       fetchImagePrimaryColor(userInfo.avatar).then((rgba) => {
-        console.log('fetchImagePrimaryColor', rgba);
       });
     }
   }, [userInfo.avatar]);
diff --git a/package.json b/package.json
index 04066ac40..7c70072ca 100644
--- a/package.json
+++ b/package.json
@@ -84,8 +84,8 @@
       "zustand": "4.3.6",
       "immer": "9.0.21",
       "mongodb": "4.2.1",
-      "mongoose@~6.0.14": "6.1.1",
-      "mongoose@^6.0.0": "6.1.1",
+      "mongoose@~6.0.14": "6.12.8",
+      "mongoose@^6.0.0": "6.12.8",
       "mongoose-findorcreate": "3.0.0",
       "@types/estree": "1.0.8",
       "@types/eslint-scope": "3.7.7"
diff --git a/server/admin/package.json b/server/admin/package.json
index 8a1011281..67d0b0a10 100644
--- a/server/admin/package.json
+++ b/server/admin/package.json
@@ -23,10 +23,10 @@
     "express": "^4.18.2",
     "express-mongoose-ra-json-server": "^0.1.0",
     "filesize": "^8.0.7",
-    "jsonwebtoken": "^8.5.1",
+    "jsonwebtoken": "^9.0.2",
     "lodash": "^4.17.21",
     "md5": "^2.3.0",
-    "mongoose": "^6.1.1",
+    "mongoose": "^6.12.8",
     "morgan": "^1.10.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
diff --git a/server/package.json b/server/package.json
index db64b6933..7f82adf82 100644
--- a/server/package.json
+++ b/server/package.json
@@ -46,7 +46,7 @@
     "got": "11.8.3",
     "i18next": "^20.3.5",
     "ioredis": "^4.27.6",
-    "jsonwebtoken": "^8.5.1",
+    "jsonwebtoken": "^9.0.2",
     "link-preview-js": "^2.1.10",
     "livekit-server-sdk": "^1.2.7",
     "lodash": "^4.17.21",
@@ -55,7 +55,7 @@
     "mkdirp": "^1.0.4",
     "moment": "^2.29.1",
     "mongodb": "4.2.1",
-    "mongoose": "6.1.1",
+    "mongoose": "6.12.8",
     "mongoose-findorcreate": "3.0.0",
     "nanoid": "^3.1.23",
     "nodemailer": "^6.7.2",
diff --git a/server/packages/sdk/package.json b/server/packages/sdk/package.json
index 0f3ce1c00..73854f75b 100644
--- a/server/packages/sdk/package.json
+++ b/server/packages/sdk/package.json
@@ -54,7 +54,7 @@
     "moleculer-repl": "^0.7.2",
     "moment": "^2.29.1",
     "mongodb": "4.2.1",
-    "mongoose": "6.1.1",
+    "mongoose": "6.12.8",
     "path-to-regexp": "^6.2.1",
     "ramda-adjunct": "^4.0.0",
     "tailchat-types": "workspace:*"
diff --git a/server/services/core/chat/message.service.ts b/server/services/core/chat/message.service.ts
index d1355138a..5a9b6cc68 100644
--- a/server/services/core/chat/message.service.ts
+++ b/server/services/core/chat/message.service.ts
@@ -178,7 +178,6 @@ class MessageService extends TcService {
         .exec(),
     ]);
 
-    console.log({ prev, next });
 
     return this.transformDocuments(ctx, {}, [...prev, message, ...next]);
   }
diff --git a/server/services/debug.service.ts b/server/services/debug.service.ts
index 0a559de61..3109f41f2 100644
--- a/server/services/debug.service.ts
+++ b/server/services/debug.service.ts
@@ -21,7 +21,6 @@ export default class TestService extends TcService {
 
   // Action
   echo(ctx: TcPureContext<{ name: string }>): string {
-    console.log(ctx.meta);
     return `Hello ${
       ctx.params.name ?? ctx.meta.t('匿名用户')
     }, \nHere is your meta info: ${JSON.stringify(ctx.meta, null, 2)}`;
diff --git a/server/test/demo/openapi-client-simple/package.json b/server/test/demo/openapi-client-simple/package.json
index 0291a6148..891e8ee51 100644
--- a/server/test/demo/openapi-client-simple/package.json
+++ b/server/test/demo/openapi-client-simple/package.json
@@ -11,7 +11,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "axios": "^0.26.0",
+    "axios": "^1.7.2",
     "express": "^4.17.2"
   },
   "devDependencies": {
diff --git a/server/test/demo/openapi-client/package.json b/server/test/demo/openapi-client/package.json
index 6b6568c76..1dffcc4dc 100644
--- a/server/test/demo/openapi-client/package.json
+++ b/server/test/demo/openapi-client/package.json
@@ -11,7 +11,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "axios": "^0.26.0",
+    "axios": "^1.7.2",
     "express": "^4.17.2",
     "openid-client": "^5.1.5"
   },
